src/legendoptics/tyvek.py

```diff
@@ -0,0 +1,46 @@
+"""
+Tyvek reflector.
+
+.. [Janacek2012] M. Janacek, "Reflectivity spectra for commonly used reflectors", https://www.osti.gov/servlets/purl/1184400
+"""
```

Comment:
```suggestion
"""Tyvek reflector."""
```

the documentation system does not like duplicate references - the same reference already exists for Tetratex.